packages/py/python-nose/package.yml

```diff
@@ -1,8 +1,9 @@
 name       : python-nose
 version    : 1.3.7
-release    : 10
+release    : 11
 source     :
     - https://pypi.python.org/packages/source/n/nose/nose-1.3.7.tar.gz : f1bffef9cbc82628f6e7d7b40d7e255aefaa1adb6a1b1d26c69a8b79e6208a98
+homepage   : https://nose.readthedocs.io/en/latest
 license    : LGPL-2.1
```

Comment:
Can you fix the license too? This is an old identifier. The new one should be either `LGPL-2.1-only` or `LGPL-2.1-or-later`, according to [SPDX](https://spdx.org/licenses/) specs.